src/AzureAuth/CommandMain.cs

```diff
@@ -251,27 +251,19 @@ public EventData AuthFlowEventData(AuthFlowResult result)
             eventData.Add("success", result.Success);
             eventData.Add("duration_milliseconds", (int)result.Duration.TotalMilliseconds);
 
-            var correlationIDs = new List<string>();
-
             if (result.Errors.Any())
             {
                 var error_messages = ExceptionListToStringConverter.Execute(result.Errors);
                 eventData.Add("error_messages", error_messages);
-                correlationIDs = ExceptionListToStringConverter.ExtractCorrelationIDsFromException(result.Errors);
             }
 
             if (result.Success)
             {
-                correlationIDs.Add(result.TokenResult.CorrelationID.ToString());
+                eventData.Add("msal_correlation_ids", result.TokenResult.CorrelationID.ToString());

```

Comment:
This key should become singular since we are only adding the 1 correlation id from the successful flow.